src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -42,6 +43,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
```

Comment:
> Maybe we're talking past each other.

Unfortunately, we are :). Sorry about that. Trying to clarify:

Reading the comments and your previous answer, I thought that you need to add this because the Primitives package under test was actually downloaded from NuGet and doesn't contain the new APIs: which, as we both agree (line 50) is not true.

Now, I am thinking that there is a reference to Requests somewhere in the FunctionalTests. That sounds odd since the FunctionalTests for Primitives shouldn't test anything that involves Requests (which also depends on Primitives).
Is this something that existed before or something that you're introducing with this PR (the test for the added properties)?

I don't see a project.json within the FunctionalTests which adds to the confusion.